src/platform/packages/shared/kbn-unified-histogram/components/chart/chart_section_template.tsx

```diff
@@ -18,22 +19,70 @@ export interface ChartSectionTemplateProps {
   id: string;
   toolbarCss?: SerializedStyles;
   toolbar?: {
+    toggleActions?: React.ReactElement;
     leftSide?: React.ReactNode;
     rightSide?: IconButtonGroupProps['buttons'];
     additionalControls?: {
       prependRight?: React.ReactNode;
     };
   };
-  children: React.ReactNode;
+  toolbarWrapAt?: 'xs' | 's' | 'm' | 'l' | 'xl' | 'xxl';
 }
 
 export const ChartSectionTemplate = ({
   id,
   toolbarCss,
   toolbar,
   children,
-}: ChartSectionTemplateProps) => {
-  const { leftSide = [], rightSide = [] } = toolbar ?? {};
+  toolbarWrapAt,
+}: React.PropsWithChildren<ChartSectionTemplateProps>) => {
+  const { toggleActions, leftSide, rightSide = [] } = toolbar ?? {};
+  const { euiTheme } = useEuiTheme();
+
+  const toolbarContainerCss = useMemo(
+    () =>
+      toolbarWrapAt
+        ? css`
+            ${toolbarCss};
+            @media (max-width: ${euiTheme.breakpoint[toolbarWrapAt]}px) {
+              min-height: auto;
+            }
+          `
+        : toolbarCss,
+    [toolbarWrapAt, euiTheme, toolbarCss]
+  );
+
+  const toolbarWrapperCss = useMemo(
```

Comment:
I'm seeing some odd stacking behaviour on the interval and breakdown selectors at smaller screen sizes:
<img src="https://github.com/user-attachments/assets/7909a001-48b0-46c7-b0a6-ee25081b2b42" />

Reply:
I'll check that. thanks

Reply:
[90253f1](https://github.com/elastic/kibana/pull/241820/commits/90253f14a68b527675d129d4eaae0096eb97a640)